src/dates/date.md

```diff
@@ -0,0 +1,209 @@
+As a beginner one of the most interesting topics you’ll learn are Dates. Even though it might sound a bit boring, it's one of the key things you might know; from your Database, API, GUI, and more learning how to deal with dates properly is a key factor in writing good applications. 
+
+# Before we start
+An important detail you must remember is Java was rushed for the web, thus you have a lot of quirky and somewhat stupid ways to do stuff. The goal of this guide will be to teach you how to use the latest Java APIs to write efficient good code. I’ll aim to teach you how not to code anything hard and rather use standardized ISO for a scalable application. 
```

Comment:
I am not even gonna disagree with this characterization, but I would rather talk about this stuff in the context of a "full history." or in the context of telling folks how social factors affect code development